Temporary workaround for bug 29668068 to handle malloc_usable_size() only
being defined in <stdlib.h> and not <malloc.h> in st_043.

Not needed for upstream, as we'll fix malloc.h to include malloc_usable_size()
before backporting to 11.4 / releasing publicly.

--- a/build/autoconf/alloc.m4
+++ b/build/autoconf/alloc.m4
@@ -23,6 +23,7 @@ MALLOC_USABLE_SIZE_CONST_PTR=const
 if test -n "$HAVE_MALLOC_H"; then
   AC_MSG_CHECKING([whether malloc_usable_size definition can use const argument])
   AC_TRY_COMPILE([#include <malloc.h>
+                  #include <stdlib.h>
                   #include <stddef.h>
                   size_t malloc_usable_size(const void *ptr);],
                   [return malloc_usable_size(0);],
